wenet/transformer/convolution.py

```diff
@@ -66,7 +66,7 @@ def __init__(self,
             self.norm = nn.BatchNorm1d(channels)
         else:
             self.use_layer_norm = True
-            self.norm = nn.LayerNorm(channels)
+            self.norm = nn.LayerNorm(channels, eps=1e-3)
```

Comment:
any explanation or comment about setting it to 1e-3?

Reply:
our model is mainly trained on fp16, and the precision is set to 1e-3 to prevent numerical problem